docs/modules/ROOT/examples/advanced-networking-interfaces.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast-1
+spec:
+  clusterSize: 3
+  repository: 'docker.io/hazelcast/hazelcast-enterprise'
+  version: 'latest'
```

Comment:
We shouldn't specify hz or mc versions explicitly. Either don't pass it or use **'{page-latest-supported-hazelcast}-slim'** (doc attribute defined in [here](https://github.com/hazelcast/hazelcast-platform-operator-docs/blob/main/docs/antora.yml#L17))